src/web-pathing.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Web Pathing Specification
+description: >
+  Specification defines a subset of possible content paths that ensures
+  compatibility with existing HTTP and Web Platform standards.
+date: 2023-11-12
+maturity: wip
+editors:
+  - name: Marcin Rataj
+    github: lidel
+    url: https://lidel.org/
+    affiliation:
+        name: Protocol Labs
+        url: https://protocol.ai/
+tags: ['architecture', 'httpGateways', 'webHttpGateways']
+---
+
+Web Pathing Specification defines a subset of possible content paths
+that ensures compatibility with existing HTTP and Web Platform standards.
+
+## Introduction
+
+TODO: Clearly explain why the specification exists, what is the problem solved here.
+
+This document specifies details of pathing for content paths that start with
+`/ipfs` and `/ipns` namespaces, and why a logical content root included in a
+content path can facilitate security isolation and relative pathing in web
+contexts.
+
+Specification includes guidance around aspects such as hash functions,
+multibases, CID versions, codecs, and how they impact implementation's ability
+to translate pathing into traversal of a DAG.
+
+The goal of this specification is to enable competing and interoperable
+implementations, all while ensuring seamless traversal of paths within the web
+ecosystem.
+
+## Specification
+
+TODO: Explain things in depth.
+The resulting specification should be detailed enough to allow competing,
+interoperable implementations.
+
+### TODO: things to cover
+
+- TODO: why it's called "web pathing": ensuring pathing is interoperable with how existing http and web platform works; covers both /ipfs and /ipns namespace semantics; defines logical content root CID that can be mapped to URL / root which enables subdomain/dnslink gateways and ipfs:// and ipns:// protocol handlers to load existing datasets, websites, and assets with relative pathing without the need for modifying them; 
+
+- TODO: how web pathing is applied to CLI Tools; path gateways; and origin contexts: subdomain/dnslink, ipfs:// ipns:// URIs
+
+- TODO: MUSTs, SHOULDs and MAYs in relation to
+
+  - TODO: multihash functions
```

Comment:
Is the intention of this section to clarify baseline multihash & codecs that must be supported to provide content for libraries such as `@helia/verified-fetch`?